src/main/java/seedu/storemando/logic/commands/ClearCommand.java

```diff
@@ -1,23 +1,59 @@
 package seedu.storemando.logic.commands;
 
 import static java.util.Objects.requireNonNull;
+import static seedu.storemando.logic.parser.CliSyntax.PREFIX_LOCATION;
+import static seedu.storemando.model.Model.PREDICATE_SHOW_ALL_ITEMS;
 
+import java.util.List;
+import java.util.function.Predicate;
+
+import seedu.storemando.logic.commands.exceptions.CommandException;
 import seedu.storemando.model.Model;
-import seedu.storemando.model.StoreMando;
+import seedu.storemando.model.item.Item;
+import seedu.storemando.model.item.LocationContainsPredicate;
 
 /**
- * Clears the storemando.
+ * Clears the storemando or a specified location.
  */
 public class ClearCommand extends Command {
 
     public static final String COMMAND_WORD = "clear";
-    public static final String MESSAGE_SUCCESS = "Inventory has been cleared!";
```

Comment:
I think we should personalize all success messages as a whole to differentiate the individual features rather than returning a generic message for all clear/list/sort command types. Maybe this would be a good place to start and we can improve on this in the coming iteration. What do the rest think?

Reply:
I think this is a good suggestion, but not a compulsory implementation since the current implementation is still a workable product. Maybe we can do this in the coming iteration.